bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java

```diff
@@ -48,14 +50,14 @@
 /**
  * Manages a pool of reusable resource resolvers and injects them into tasks
  */
-class ActionManagerImpl implements ActionManager {
+class ActionManagerImpl extends CancelHandler implements ActionManager {
 
-    private static final Logger LOG = LoggerFactory.getLogger(ActionManagerImpl.class);
+    transient private static final Logger LOG = LoggerFactory.getLogger(ActionManagerImpl.class);
```

Comment:
In some places I have to make the logger transient because of GSON but it makes sense to make sure that the class is serializable so that it is more clear what is happening.   Other places that field is transient in classes that are not serialized ever so it makes so sense.  I've done a pass through and cleaned that up.  The PR will be updated shortly.